sodiumpp/include/sodiumpp/locked_string.h

```diff
@@ -22,13 +22,13 @@ class locked_string final
              * for move construction for std::string use static methods:
              * move_from_locked_string() and move_from_not_locked_string
              */
-            explicit locked_string(std::string && str) noexcept; ///< only move, not call mlock()
+            //explicit locked_string(std::string && str) noexcept; ///< only move, not call mlock()
     public:
             locked_string() = default;
             locked_string(const locked_string &);
-            locked_string(locked_string &&) = default;
+            locked_string(locked_string &&other);
```

Comment:
... or actually move to custom made trivial "vector" to have guarantee that address is same across moving our locked  "string"